mypy/errorcodes.py

```diff
@@ -135,6 +135,10 @@ def __str__(self) -> str:
 )
 UNSAFE_VARIANCE: Final = ErrorCode("unsafe-variance", "Incorrect usages of variance", "General")
 UNUSED_IGNORE: Final = ErrorCode("unused-ignore", "Ignore comment is unused", "General")
+
+DYNAMIC_TYPING: Final = ErrorCode("dynamic", "Usage of dynamic typing", "General")
```

Comment:
i think its confusing that this error code refers to `Any` as `dynamic`, i think it should clarify what `dynamic` means
```suggestion
DYNAMIC_TYPING: Final = ErrorCode("dynamic", "Usage of dynamic typing (Any)", "General")
```